rllib/policy/tests/test_policy_checkpoint_restore.py

```diff
@@ -103,7 +101,22 @@ def test_restore_checkpoint_with_nested_obs_space(self):
             space.original_space = gym.spaces.Discrete(2)
             space = space.original_space
 
-        policy = PPOTorchPolicy(obs_space, gym.spaces.Discrete(2), MODEL_DEFAULTS)
+        class NestedObsSpaceDummyEnv(gym.Env):
```

Comment:
This is not needed, right? You can directly pass obs_space, and action_space to config without an env?
```
policy = PPOConfig().environment(observation_space=obs_space, action_space=...).build().get_policy()
```

If env started complaining, turn the env checker off.